frontend/common/src/components/form/Select/SelectFieldV2.test.tsx

```diff
@@ -210,8 +210,6 @@ describe("SelectFieldV2", () => {
       fireEvent.submit(screen.getByRole("button"));
     });
     expect(mockSubmit).not.toBeCalled();
-    expect(screen.queryByRole("alert")).toBeInTheDocument();
```

Comment:
So, good thing you mentioned this. Turns out for **_some_** reason, `react-select` already has an `aria-describedby` attribute that points to a `div` which acts as the placeholder rather than a `placeholder` 😢 

Going to see if I can figure this out.

<img width="1512" alt="Screenshot 2022-11-23 at 8 14 20 AM" src="https://user-images.githubusercontent.com/4127998/203557036-286913c6-dacc-401f-b6e3-fd8630ff217e.png">

Reply:
Thanks to @JamesHuf on his work with the `trackUnsaved` prop, I was able to sort this out 😁 

Fixed the issue along with adding a test in [fix select v2 description](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4818/commits/b9f460158d305caa078a573c5db4c23f7f24a404)